pkg/controllers/nodeclaim/lifecycle/liveness.go

```diff
@@ -55,7 +55,7 @@ func (l *Liveness) Reconcile(ctx context.Context, nodeClaim *v1beta1.NodeClaim)
 	if err := l.kubeClient.Delete(ctx, nodeClaim); err != nil {
 		return reconcile.Result{}, client.IgnoreNotFound(err)
 	}
-	logging.FromContext(ctx).With("ttl", registrationTTL).Debugf("terminating due to registration ttl")
+	log.FromContext(ctx).V(0).WithValues("ttl", registrationTTL).Info("terminating due to registration ttl")
```

Comment:
```suggestion
	log.FromContext(ctx).V(1).WithValues("ttl", registrationTTL).Info("terminating due to registration ttl")
```

I'm assuming this is what you meant

Reply:
Yes, good catch!